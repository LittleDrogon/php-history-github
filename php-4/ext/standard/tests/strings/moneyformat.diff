001- string(7) "X$3.14Y"
001+ string(6) "X3.14Y"
003- Warning: money_format() [/phpmanual/function.money-format.html]: Only a single %ci or %cn token can be used in %s on line %d
003+ Warning: money_format() [/phpmanual/function.money-format.html]: Only a single %i or %n token can be used in /home/star/Desktop/php-history/php-4/ext/standard/tests/strings/moneyformat.php on line 4