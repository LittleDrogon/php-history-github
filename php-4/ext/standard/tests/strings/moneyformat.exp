string(7) "X$3.14Y"

Warning: money_format() [/phpmanual/function.money-format.html]: Only a single %ci or %cn token can be used in %s on line %d
bool(false)