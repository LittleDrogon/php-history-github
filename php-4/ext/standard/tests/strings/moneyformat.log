
---- EXPECTED OUTPUT
string(7) "X$3.14Y"

Warning: money_format() [/phpmanual/function.money-format.html]: Only a single %ci or %cn token can be used in %s on line %d
bool(false)
---- ACTUAL OUTPUT
string(6) "X3.14Y"

Warning: money_format() [/phpmanual/function.money-format.html]: Only a single %i or %n token can be used in /home/star/Desktop/php-history/php-4/ext/standard/tests/strings/moneyformat.php on line 4
bool(false)
---- FAILED
