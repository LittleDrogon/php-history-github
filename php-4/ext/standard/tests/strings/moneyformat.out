string(6) "X3.14Y"

Warning: money_format() [/phpmanual/function.money-format.html]: Only a single %i or %n token can be used in /home/star/Desktop/php-history/php-4/ext/standard/tests/strings/moneyformat.php on line 4
bool(false)